src/libcommonserver/log/log.cpp

```diff
@@ -26,15 +26,15 @@
 
 #include <codecvt>
 
-#define LOGGER_INSTANCE_NAME L"Main"
-#define LOGGER_APP_RF_NAME L"RollingFileAppender"
-#define LOGGER_APP_RF_PATTERN L"%D{%Y-%m-%d %H:%M:%S:%q} [%-0.-1p] (%t) %b:%L - %m%n"
-#define LOGGER_APP_RF_MAX_FILE_SIZE 500  // MB
-#define LOGGER_APP_RF_MAX_BACKUP_IDX 4
-#define LOGGER_APP_PURGE_AFTER_HOURS 24
-
 namespace KDC {
 
+const std::wstring Log::instanceName = L"Main";
+const std::wstring Log::rfName = L"RollingFileAppender";
+const std::wstring Log::rfPattern = L"%D{%Y-%m-%d %H:%M:%S:%q} [%-0.-1p] (%t) %b:%L - %m%n";
+const int Log::rfMaxFileSize = 500;  // MB
+const int Log::rfMaxBackupIdx = 4;   // Max number of backup files
+const int Log::logsPurgeRate = 7;    // days
```

Comment:
Should it be a common setting shared across client and server ?

Reply:
OK, I moved some shared constants to CommonUtility class